src/Tribe/Attendees_Table.php

```diff
@@ -1086,4 +1086,17 @@ public function search_box( $text, $input_id ) {
 
 		echo $search_box;
 	}
+
+	/**
+	 * Handler for the Edit Attendee column
+	 *
+	 * @param $item
+	 *
+	 * @since TBD
+	 *
+	 * @return string
```

Comment:
:no_entry_sign: Function return type is not void, but function has no return statement

<sub>hash: 276e5538ef6e62c7b681c0954f000ce1</sub>